addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -21,6 +24,18 @@
             <div data-name="solar" title="Promotion Program" data-img="/mass_mailing_themes/static/src/img/theme_imgs/solar_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
                 <t t-call="mass_mailing_themes.theme_solar_template"/>
             </div>
+            <div data-name="coffeebreak" title="Coffee break" data-layout-styles="background-color: rgb(238, 233, 226) !important;" data-img="/mass_mailing_themes/static/src/img/theme_imgs/coffee_break_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
+                <t t-call="mass_mailing_themes.theme_coffeebreak_template"/>
+            </div>
+            <div data-name="roadshow1" title="Roadshow Schedule" data-layout-styles="background-color: rgb(255, 255, 255) !important;" data-img="/mass_mailing_themes/static/src/img/theme_imgs/roadshow_schedule_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
+                <t t-call="mass_mailing_themes.theme_roadshow1_template"/>
+            </div>
+            <div data-name="roadshow2" title="Roadshow Follow-up" data-layout-styles="background-color: rgb(255, 255, 255) !important;" data-img="/mass_mailing_themes/static/src/img/theme_imgs/roadshow_followup_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
+                <t t-call="mass_mailing_themes.theme_roadshow2_template"/>
+            </div>
+            <div data-name="graphic" title="Training" data-layout-styles="background-color: rgb(220, 234, 255) !important;" data-img="/mass_mailing_themes/static/src/img/theme_imgs/training_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
```

Comment:
Same for the three above here. The name of the thumbnail does not seem to match the `data-name` of the theme. It's no big deal IMO but I just wanted to point it out in case it was a mistake or so. 🤷

Reply:
I fixed all the template names to make everything more coherent 👍

Reply:
@anso-odoo no, no need to modify the PO files, that's the job of Transifex. As you are working in master, even updating the .pot is not required (but possible).